new_repository.md

```diff
@@ -1,26 +1,43 @@
 # Setting up a new repository
 
-After you have a GitHub repository created, the next step is usually to initiate a
-`README.md`, add an MIT license, and a `.gitignore` file, unless these things were added
-automatically via the GitHub web interface. Following that, you may want to set up
+By default, all our repositories should be public, unless there's a specific reason
+that's not possible.
+
+After you have created a GitHub repository, the next step is to initiate a
+`README.md`, add an MIT license, and a `.gitignore` file, unless these things were
+added automatically via the GitHub web interface.
```

Comment:
So basically the initial commit of these files should be on the `main` branch, then change the GitHub repo settings. Got it.

Reply:
That's a good point, arguably even those ideally should be reviewed. Doesn't matter too much for the initial skeleton files though. I've changed the order.